packages/editor/src/store/actions.js

```diff
@@ -47,7 +64,6 @@ export function resetPost( post ) {
  * post has been received, by initialization or autosave.
  *
  * @param {Object} post Autosave post object.
- *
```

Comment:
From which guideline are you asserting the documentation standard proposed here?

Elsewhere, the `@param` and `@return` are treated as separately grouped, within each group aligned on the name and descriptions, as documented here:

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions

There are multiple instances elsewhere in the pull request.

Reply:
Huh, I've honestly never noticed this before.  Will correct.

Reply:
So turns out my IDE (phpStorm) doesn't have much ability to customize the jsdocs template it creates.  So I've just never noticed that its defaults diverge so much from the wp core standards (and thus in my own projects I've just never thought anything of it).  I'll try to watch for this in the future.